{{cookiecutter.project_slug}}/Makefile

```diff
@@ -30,6 +30,9 @@ CUSTOM_ENV_NAME?=image:neuromation-$(PROJECT_POSTFIX)
 
 ##### VARIABLES YOU MAY WANT TO MODIFY #####
 
+# jupyter mode. Available options: notebook, lab.
```

Comment:
Jupyter mode. Available options: notebook (to run Jupyter Notebook), lab (to run JupyterLab).